src/VisionModel/googlenet.jl

```diff
@@ -0,0 +1,59 @@
+using Flux
+
+export googlenet
+
+function conv_block(kernelsize::Tuple{Int64,Int64}, inplanes::Int64, outplanes::Int64; stride::Int64=1, pad::Int64=0)
+  conv_layer = []
+  push!(conv_layer, Conv(kernelsize, inplanes => outplanes, stride = stride, pad = pad)) 
+  push!(conv_layer, BatchNorm(outplanes, relu))
+  return conv_layer
+end
+
+function inceptionblock(inplanes, out_1x1, red_3x3, out_3x3, red_5x5, out_5x5, pool_proj)
+  branch1 = conv_block((1,1), inplanes, out_1x1)
+
+  branch2 = Chain(conv_block((1,1), inplanes, red_3x3),
+             conv_block((3,3), red_3x3, out_3x3; pad=1))        
+
+  branch3 = Chain(conv_block((1,1), inplanes, red_5x5),
+             conv_block((5,5), red_5x5, out_5x5; pad=1)) 
+
+  branch4 = Chain(MaxPool((3, 3), stride=1, pad=1),
+             conv_block((1,1), inplanes, pool_proj))
+
+  inception_layer = x -> begin
+    y1 = branch1[1](x)
+    y2 = branch2[2][1](branch2[1][1](x))
+    y3 = branch3[2][1](branch3[1][1](x))
+    y4 = branch4[2][1](branch4[1](x))
```

Comment:
No it should be
```julia
branch1 = Chain(conv_block((1, 1), inplanes, out_1x1)...)
# later
y1 = branch1(x)
```
Same for all the others.

Reply:
This gives ``LoadError : syntax: "..." expression outside call``

Reply:
Are you sure you are doing it correctly? We do the exact same thing on L37+.

Reply:
You can commit the broken stuff so I can see.

Reply:
Its working now. not sure how I got that error previously. But now, it is giving that dimension mismatch error. ``ERROR: DimensionMismatch("mismatch in dimension 1 (expected 28 got 26)")``

Reply:
It's actually working similar to the change I made previously.

Reply:
Can you push your code? It will be easier if I can see the error on the CI.